pkg/asset/images.go

```diff
@@ -8,6 +8,6 @@ var DefaultImages = ImageVersions{
 	Calico:          "quay.io/calico/node:v3.0.3",
```

Comment:
This is quite an old image and idea why we are tracking so far behind?

Reply:
calico updates have not been considered

Reply:
That is being addressed here: https://github.com/kubernetes-sigs/bootkube/pull/1074 :-D (draft PR for now :))